thrust/iterator/transform_input_output_iterator.h

```diff
@@ -159,4 +161,3 @@ make_transform_input_output_iterator(Iterator io, InputFunction input_function,
  */
 
 THRUST_NAMESPACE_END
-
```

Comment:
I think we should keep the empty line a the end. Could you bring them back?

Reply:
Do we really need TWO empty lines at the end? There were two before, now there is one. github hides a single empty line at the end of a file. If there is no empty line, it displays a warning character. However, for the sake of RAPIDS' patch file, I will put back the redundant empty line.

(I didn't even realize I made this change, FWIW. I thought I undid all cosmetic changes.)

Reply:
Ah ok then the Github UI failed me.

Please go ahead and scratch that comment

Reply:
For the sake of simplifying cuDF's patch file, I put them back.